### PR TITLE
Tokens: Update Typography Tokens

### PR DESCRIPTION
**Pull request by anechol**
# Description

Updates the following tokens:
- Font Weight (updated values)
- Font Family (Greet -> GreetStandard)
- Line Height (new value, 1.42)

## Type of change

- [x] Tokens (created, updated, or removed design system tokens)

# How Has This Been Tested?

- [ ] unit tests
- [ ] e2e tests
- [ ] accessibility tests
- [x] tested manually
- [ ] other:

**Test Configuration**:

- Pine versions:
- OS:
- Browsers:
- Screen readers:
- Misc:

### CODE REVIEW

File: .github/workflows/actions/publish-npm/action.yml [hunk 1]
```diff
@@ -35,7 +35,6 @@ runs:
         registry-url: 'https://registry.npmjs.org'
 
     - name: Install Dependencies
-      if: inputs.version != ''
```
Comment (ju-Skinner) [inline .github/workflows/actions/publish-npm/action.yml]: looks like you created this branch against `main` and not `mercury-rising`.

File: .release-please-manifest.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-  "libs/core": "1.0.4",
-  "libs/react": "1.0.4",
-  "libs/doc-components": "1.0.0",
```
Comment (ju-Skinner) [inline .release-please-manifest.json]: same as first comment up top.

File: .github/workflows/release-please.yml [hunk 3]
```diff
@@ -76,7 +76,7 @@ jobs:
           git config user.email "dev+github-bot@kajabi.com"
 
       - name: Build Packages
-        run: npm install --legacy-peer-deps
```
Comment (ju-Skinner) [inline .github/workflows/release-please.yml]: same as first comment up top.

File: libs/core/src/global/styles/tokens/core/_core.scss [hunk 4]
```diff
@@ -2,6 +2,8 @@
   --pine-border-radius-0: 0;
   --pine-border-radius-50: 4px;
   --pine-border-radius-75: 6px;
+  --pine-border-radius-50: 4px;
+  --pine-border-radius-75: 6px;
```
Comment (pixelflips) [inline libs/core/src/global/styles/tokens/core/_core.scss]: these seem to be duplicates

File: libs/core/src/global/styles/tokens/core/_core.scss [hunk 5]
```diff
@@ -146,6 +149,7 @@
   --pine-line-height-300: 3;
   --pine-spacing-0: 0;
   --pine-spacing-50: 4px;
+  --pine-spacing-50: 4px;
```
Comment (pixelflips) [inline libs/core/src/global/styles/tokens/core/_core.scss]: these seem to be duplicates.

**Review by pixelflips — CHANGES_REQUESTED** [review]
there are a few duplicates in the `_core.scss` file

File: libs/core/src/global/styles/tokens/core/_core.scss (hunk 5 above)
Comment (anechol) [reply libs/core/src/global/styles/tokens/core/_core.scss]: This should be fixed now.

**Review by pixelflips — COMMENTED** [review]
I think there are some file conflicts going on.

File: libs/core/src/global/styles/tokens/core/_core.scss [hunk 6]
```diff
@@ -119,5 +162,14 @@
...
+  --pine-letter-spacing-0: 0px;
+  --pine-letter-spacing-78: 0.11px;
+  --pine-letter-spacing-85: 0.12px;
+  --pine-letter-spacing-100: 0.14px;
+  --pine-letter-spacing-114: 0.16px;
+  --pine-letter-spacing-128: 0.18px;
+  --pine-letter-spacing-142: 0.20px;
+  --pine-letter-spacing-157: 0.22px;
+  --pine-letter-spacing-171: 0.24px;
+  --pine-letter-spacing-185: 0.26px;
```
Comment (ju-Skinner) [inline libs/core/src/global/styles/tokens/core/_core.scss]: I'm not sure how I feel about this. lol

File: libs/core/src/global/styles/tokens/core/core.json [hunk 7]
```diff
@@ -153,311 +153,483 @@
...
+    },
+    "85": {
+      "value": "12px",
+      "type": "fontSizes"
+    },
     "100": {
+      "value": "14px",
+      "type": "fontSizes"
+    },
+    "116": {
```
Comment (ju-Skinner) [inline libs/core/src/global/styles/tokens/core/core.json]: This should be 114

File: libs/core/src/global/styles/tokens/semantic/semantic.json [hunk 8]
```diff
@@ -238,171 +238,5 @@
...
-    "type": "typography"
-  },
-  "body-xs-bold": {
-    "value": {
-      "fontWeight": "{font-weight.bold}",
-      "fontSize": "{font-size.075}",
-      "lineHeight": "{line-height.100}",
-      "fontFamily": "{font-family.circular}"
-    },
-    "type": "typography"
```
Comment (ju-Skinner) [inline libs/core/src/global/styles/tokens/semantic/semantic.json]: Is there a reason why we removed these?

**Review by ju-Skinner — CHANGES_REQUESTED** [review]
I am requesting changes due to the key for 16px font to be `114` vs `116`.

**Review by ju-Skinner — APPROVED** [review]
👏🏾

File: libs/core/src/global/styles/tokens/semantic/semantic.json (hunk 8 above)
Comment (anechol) [reply libs/core/src/global/styles/tokens/semantic/semantic.json]: These were removed in a previous PR because I am redoing semantic tokens.

**Review by pixelflips — APPROVED** [review]
LGTM! Great work! 👍🏼

**Pull request merged**